src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -170,7 +173,7 @@ export function PreSubmitSection(props) {
             <Checkbox
               id="veteran-certify"
               name="veteran-certify"
-              label="I certify the information above is correct and true to the best of my knowledge and belief."
+              label={sotCheckboxLabel}
```

Comment:
Could you move the logic for the `sotCheckboxLabel` down to here? It should function the same, but save defining a constant that isn't re-used anywhere else. It'll also match the other statement of truth customizations.

`label=statementOfTruth?.checkboxLabel || 'I certify the information above is correct and true to the best of my knowledge and belief.'`

Reply:
Done and thx!